doc/howto/write_a_view/index.md

```diff
@@ -0,0 +1,429 @@
+# How to write a view {#howto_write_a_view}
+
+[TOC]
+
+This HowTo documents how to write a view using the standard library and some helpers from SeqAn3.
+
+\tutorial_head{Difficult, 120 min, \ref tutorial_concepts\,\ref tutorial_ranges, }
+
+\note
+Some of the links from this HowTo only resolve in the developer documentation, because
+they referred to entities from the seqan3::detail namespace.
+We recommend you open this tutorial from http://docs.seqan.de/seqan/3.0.0-master-dev/
+\endnote
```

Comment:
remove

Reply:
why?

Reply:
`\endnote` doesn't exist.
You should also see a warning about it.

Reply:
Hm, that's confusing, why do the other all have \end but not \note? Or is note not a macro defined by us?

Reply:
@smehringer is responsible for this heresy

Reply:
![image](https://user-images.githubusercontent.com/7955878/55020828-f526fc00-4ff7-11e9-8e6e-4dd1526696a8.png)

no I am not

Reply:
Avenja means to say that this is a doxygen macro.